web-ui/src/components/team-results/EditTeamModal.jsx

```diff
@@ -179,7 +179,7 @@ const EditTeamModal = ({ team = {}, open, onSave, onClose, headerText }) => {
             label="Active"
             variant="outlined"
             className="halfWidth"
-            checked={editedTeam.active ? editedTeam.active : false}
+            checked={editedTeam.active}
```

Comment:
At first, I was absolutely terrified that I was the one responsible for this. My relief was immense.